src/content/docs/new-relic-solutions/observability-maturity/operational-efficiency/data-governance-optimize-ingest-guide.mdx

```diff
@@ -0,0 +1,1624 @@
+---
+title: "Optimize your data ingest"
+tags:
+  - Observability maturity
+  - Operational efficiency 
+  - Data ingest cost
+  - Sampling rate
+  - Drop rules
+  - Observability as code
+  - Value drivers
+  - Bill and Usage Data
+  - Data ingest cost
+metaDescription: "The third part of New Relic's data ingest governance series on optimizing how you ingest and use your telemetry data: this focuses on optimizing data ingest."
+redirects:
+  - /docs/telemetry-data-platform/manage-data/data-governance
+  - /docs/data-governance
+  - /docs/telemetry-data-platform/get-started/manage-data/data-governance
+  - /telemetry-data-platform/get-started/manage-data/data-governance
+  - /docs/new-relic-solutions/observability-maturity/operational-efficiency/dg-optimizing
+freshnessValidatedDate: never
+---
+
+import omaoedgOptimizingIcon from 'images/oma-oe-dg_icon_optimizing.webp'
+
+import omaoedgValueDriverUptime from 'images/oma-oe-dg_diagram_value-driver-uptime.webp'
+
+import omaoedgValueDriverCustomer from 'images/oma-oe-dg_diagram_value-driver-customer.webp'
+
+import omaoedgValueDriverInnovation from 'images/oma-oe-dg_diagram_value-driver-innovation.webp'
+
+import omaoedgKubeStateMetrics from 'images/oma-oe-dg_diagram_update-k8s-kube-state-metrics.webp'
+
+import omaoedgKubernetesScrapeInterval from 'images/oma-oe-dg_diagram_update-k8s-scrape-interval.webp'
```

Comment:
Hi @DavSanchez ! We're no longer using import statements like these. You can just refer to the image path in the `static/images` directory directly in the `src`

Reply:
Thank you @clarkmcadoo. I've fixed the import statements in both files.